tds-item-selection-components/src/main/java/tds/itemselection/selectors/impl/AdaptiveSelector2013.java

```diff
@@ -41,11 +41,7 @@
   private static Logger logger = LoggerFactory.getLogger(AdaptiveSelector2013.class);
 
   // a combination of test-constant and examinee-variable data
-  private Blueprint blueprint;
-  private Cset1 cset1;
-  private ItemCandidatesData itemCandidates;
-  private TestSegment segment;
-  private Random rand = new Random();
+  private final Random rand = new Random();
```

Comment:
After a side discussion with @wtritch I'm moving this from an instance level variable to the methods that need it.